config/config.go

```diff
@@ -74,7 +75,10 @@ type Config struct {
 	PSK                pnet.PSK
 
 	RelayCustom bool
-	Relay       bool
+	Relay       bool // should the relay transport be used
+
+	DisableRelayService bool // should we run a circuitv2 relay (if publicly reachable)
```

Comment:
let's heed to @stebalien's request to not enable it by default, so this should be `EnableRelayService` I guess and the logic inverted.

Reply:
Makes sense.
We'll still enable this by default in IPFS / Filecoin, won't we?

Reply:
>  We'll still enable this by default in IPFS / Filecoin, won't we?

Yes, or at least in IPFS. But libp2p is a library not an application.